### DISCUSSION TITLE
How to call a method with multiple types in an array list

### DISCUSSION THREAD

**Discussion by prlksk** [FreeOpcUa/python-opcua#1544] [Q&A]
I have such a class: 
      class Body(object):
          #nested type
          
          ua_types = [
              ('profile', 'InstagramProfile'),
              ('follower', 'InfluencerPhotos'),
                  ]

          def __init__(self):
              self.profile =  InstagramProfile.3M_Followers
              self.follower = InfluencerPhotos.Random_vacation_34

I defined Instagramprofile and InfluencerPhotos as separete intenum typed classes. But I get an error of different types so ua cant guess the type: in _guess_type raise UaError("List of multiple types. Could not guess UA type of variable {0}".format(error_val)) opcua.ua.uaerrors._base.UaError: List of multiple types. Could not guess UA type of variable

**Comment by prlksk** [comment]
I also tried these:           my_list : List[Union[IntEnum, object]] =[[p1.profile.3M_Followers, p1.follower.Random_vacation_34]]
  result5 = dvideo_influence.call_method(mvideo_influence, my.list)
Nothing is working how can I fix this?

**Reply by schroeder-** [reply]
You can't use a python class. Are you on the server or client?

For Client do this:

```
client.load_type_definitions()
basic_msg = get_ua_class('Body')()
basic_msg.follower = p1.profile.3M_Followers
basic_msg.profile = p1.follower.Random_vacation_34
basic_var.set_value(basic_msg)
```
On the server see this example:

examples/server-create-custom-structures.py
